netci.groovy

```diff
@@ -209,67 +213,70 @@ def buildArchConfiguration = ['Debug': 'x86',
+                        }
                     }
                 }
-            }
 
-            // Set the affinity.  OS name matches the machine affinity.
-            if (osName == 'Windows_NT' || osName == 'OSX10.12') {
-                Utilities.setMachineAffinity(newJob, osForMachineAffinity, "latest-or-auto-elevated")
-            }
-            else if (osGroupMap[osName] == 'Linux') {
-                Utilities.setMachineAffinity(newJob, osForMachineAffinity, 'outer-latest-or-auto')
-            } else {
-                Utilities.setMachineAffinity(newJob, osForMachineAffinity, 'latest-or-auto');
-            }
+                // Set the affinity.  OS name matches the machine affinity.
+                if (osName == 'Windows_NT' || osName == 'OSX10.12') {
+                    Utilities.setMachineAffinity(newJob, osForMachineAffinity, "latest-or-auto-elevated")
+                }
+                else if (osGroupMap[osName] == 'Linux') {
+                    Utilities.setMachineAffinity(newJob, osForMachineAffinity, 'outer-latest-or-auto')
+                } else {
+                    Utilities.setMachineAffinity(newJob, osForMachineAffinity, 'latest-or-auto');
+                }
 
-            // Set up standard options.
-            Utilities.standardJobSetup(newJob, project, isPR, "*/${branch}")
-            // Add the unit test results
-            Utilities.addXUnitDotNETResults(newJob, 'bin/**/testResults.xml')
-            // Add archival for the built data.
-            Utilities.addArchival(newJob, "msbuild.log", '', doNotFailIfNothingArchived=true, archiveOnlyIfSuccessful=false)
-            // Set up appropriate triggers.  PR on demand, otherwise nightly
-            if (isPR) {
-                // Set PR trigger.
-                // TODO: More elaborate regex trigger?
-                Utilities.addGithubPRTriggerForBranch(newJob, branch, "OuterLoop ${osName} ${configurationGroup} ${archGroup}", "(?i).*test\\W+outerloop\\W+${osName}\\W+${configurationGroup}.*")
-            }
-            else {
-                // Set a periodic trigger
-                Utilities.addPeriodicTrigger(newJob, '@daily')
+                // Set up standard options.
+                Utilities.standardJobSetup(newJob, project, isPR, "*/${branch}")
+                // Add the unit test results
+                Utilities.addXUnitDotNETResults(newJob, 'bin/**/testResults.xml')
+                // Add archival for the built data.
+                Utilities.addArchival(newJob, "msbuild.log", '', doNotFailIfNothingArchived=true, archiveOnlyIfSuccessful=false)
+                // Set up appropriate triggers.  PR on demand, otherwise nightly
+                if (isPR) {
+                    // Set PR trigger.
+                    // TODO: More elaborate regex trigger?
+                    def testRunName = targetGroup == 'netfx' ? 'netfx' : osName
```

Comment:
I did it here this way so that it is easier to remember as it is for the regex to trigger a test through dotnet bot test ..... this -- anyway do you want to change it to include the target group and os name here as well?